Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -430,6 +429,7 @@ extension SwiftLanguageService {
     let req = sourcekitd.dictionary([
       keys.request: self.requests.editorClose,
       keys.name: uri.pseudoPath,
+      keys.cancelBuilds: 0,
```

Comment:
Wasn't the intention to do it for `reopenDocument`? (or should we do it for both?)

Reply:
Yeah, I'd say we really only want for re-open